plugins/source/postgresql/client/list_tables.go

```diff
@@ -31,13 +38,15 @@ SELECT
 		ELSE false
 	END AS not_null,
     CASE
-		WHEN conkey IS NOT NULL AND (contype = 'p' OR contype = 'u') AND array_position(conkey, pg_attribute.attnum) > 0 THEN true
+		WHEN conkey IS NOT NULL AND contype = 'u' AND array_position(conkey, pg_attribute.attnum) > 0 THEN true
```

Comment:
I think we want to return `is_unique` when the constraint a primary key, as that implicitly says the column is unique

Reply:
Maybe the fix should be to update the query to mark the column as unique only if there's a single primary key